presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -247,6 +248,9 @@ protected void setup(Binder binder)
         // column properties
         binder.bind(ColumnPropertyManager.class).in(Scopes.SINGLETON);
 
+        // analyze properties
+        binder.bind(AnalyzePropertyManager.class).in(Scopes.SINGLETON);
```

Comment:
Technically, this is needed only on the coordinator, right?

Reply:
Yes. But we also do the same for other property managers (e.g. `TablePropertyManager`, `ColumnPropertyManager`). Should have a commit to fix them all together.